specs/trip_dispatch_spec.rb

```diff
@@ -88,5 +89,61 @@
       passenger.must_be_instance_of RideShare::Passenger
       passenger.trips.must_include trip
     end
+
+  end
+
+  describe "request_trip method" do
```

Comment:
What about:
1.  When all the drivers are unavailable.
1.  The passenger doesn't exist